src/commands/lambda/flare.ts

```diff
@@ -53,44 +53,44 @@ export class LambdaFlareCommand extends Command {
     this.context.stdout.write(flareRenderer.renderLambdaFlareHeader(this.isDryRun))
 
     // Validate function name
+    const errorMessages: string[] = []
     if (this.functionName === undefined) {
       this.context.stderr.write(commonRenderer.renderError('No function name specified. [-f,--function]'))
 
       return 1
     }
```

Comment:
Note that I immediately print and return when `this.functionName` is undefined because this variable is used to calculate `region` and must be defined before continuing.